requirements.in

```diff
@@ -25,6 +25,3 @@ scipy
 boto
 boto3
 semantic_version
-# sentiment analysis
-google-cloud
-joblib
```

Comment:
And remove [test_sentiment_scorer.py](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/tests/workers/test_sentiment_scorer.py) please.